DAO.sol

```diff
@@ -744,9 +744,10 @@ contract DAO is DAOInterface, Token, TokenCreation {
 
         reward = rewardAccount.balance < reward ? rewardAccount.balance : reward;
 
+        paidOut[_account] += reward;
         if (!rewardAccount.payOut(_account, reward))
             throw;
```

Comment:
If it failed, the transaction would throw, so this would be as if it never happened.